reference/git_github.rst

```diff
@@ -3,17 +3,6 @@ Git and Github
 
 This document describes some best practices for using Git and Github.
 
-Best practises
```

Comment:
Could this be kept here using an include? It's still relevant here and it's not so bad having the same info twice if it's an include.